Administrator Guide/GlusterFS Introduction.md

```diff
@@ -1,63 +1,33 @@
-Introducing Gluster File System
-===============================
-
-GlusterFS is an open source, distributed file system capable of scaling to
-several petabytes and handling thousands of clients. It is a file system with
-a modular, stackable design, and a unique no-metadata server architecture.
-This no-metadata server architecture ensures better performance,
-linear scalability, and reliability. GlusterFS can be
-flexibly combined with commodity physical, virtual, and cloud resources
-to deliver highly available and performant enterprise storage at a
-fraction of the cost of traditional solutions.
-
-GlusterFS clusters together storage building blocks over Infiniband RDMA
-and/or TCP/IP interconnect, aggregating disk and memory resources and
-managing data in a single global namespace.
-
-GlusterFS aggregates various storage servers over network interconnects
-into one large parallel network file system. Based on a stackable user space
-design, it delivers exceptional performance for diverse workloads and is a key
-building block of GlusterFS.
-The POSIX compatible GlusterFS servers, use any ondisk file system which supports
-extended attributes (eg: ext4, XFS, etc) to format to store data on disks, can be
-accessed using industry-standard access protocols including Network File System (NFS)
-and Server Message Block (SMB).
+What is Gluster ?
+=================
 
-![640px-glusterfs_architecture](https://cloud.githubusercontent.com/assets/10970993/7412187/dbb94286-ef5d-11e4-93ca-d4959a94db85.png)
+Gluster is a scalable, distributed file system that aggregates disk and memory resources into a single global namespace.
 
-GlusterFS is designed for today's high-performance, virtualized cloud
-environments. Unlike traditional data centers, cloud environments
-require multi-tenancy along with the ability to grow or shrink resources
-on demand. Enterprises can scale capacity, performance, and availability
-on demand, with no vendor lock-in, across on-premise, public cloud, and
-hybrid environments.
+### Advantages
 
-GlusterFS is in production at thousands of enterprises spanning media,
-healthcare, government, education, web 2.0, and financial services.
+  * Scales to several petabytes
+  * Handles thousands of clients
+  * POSIX compatible
+  * Uses commodity hardware
+  * Can use any ondisk filesystem that supports extended attributes
+  * Accessible using industry standard protocols like NFS and SMB
+  * Provides replication, quotas, geo-replication, snapshots and bitrot detection
+  * Allows optimization for different workloads
+  * Open Source
 
-## Commercial offerings and support ##
 
-Several companies offer support or [consulting](http://www.gluster.org/consultants/).
+![640px-glusterfs_architecture](https://cloud.githubusercontent.com/assets/10970993/7412187/dbb94286-ef5d-11e4-93ca-d4959a94db85.png)
```

Comment:
Can this link be relative instead ?

Reply:
Do you mean copy it into the project?

Reply:
If it's not in the repo, there's nothing to be done here.

Reply:
You can add it to the repo, provided we know the source and its usage license.